examples/imagenet/models.py

```diff
@@ -131,5 +132,21 @@ def __call__(self, x, train: bool = True):
                     block_cls=BottleneckResNetBlock)
 
 
+ResNet18Local = partial(ResNet, stage_sizes=[2, 2, 2, 2],
+                        block_cls=ResNetBlock, conv=nn.ConvLocal)
+ResNet34Local = partial(ResNet, stage_sizes=[3, 4, 6, 3],
+                        block_cls=ResNetBlock, conv=nn.ConvLocal)
+ResNet50Local = partial(ResNet, stage_sizes=[3, 4, 6, 3],
```

Comment:
Is there a paper link for these networks? Intuitively I would guess these networks have a huge number of params? I think even the first conv is like 100M+ params?

Reply:
Good point - removed ResNets with depth > 18 that aren't tested (_ResNet1 and ResNet18 are tested).

Re references, AFAIK first mentions are the old results in
https://link.springer.com/article/10.1007/BF00342633
http://yann.lecun.com/exdb/publis/pdf/lecun-89.pdf

But I am not aware of Locally-connected resnets specifically, and added these only for the purpose of simple examples / tests.